recipes/pytest-xprocess/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "0.18.1" %}
+{% set cov_fail_under = 90 %}
+
+package:
+  name: pytest-xprocess
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/p/pytest-xprocess/pytest-xprocess-{{ version }}.tar.gz
+  sha256: fd9f30ed1584b5833bc34494748adf0fb9de3ca7bacc4e88ad71989c21cba266
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.5
+    - setuptools-scm
+  run:
+    - psutil
+    - pytest >=2.8
+    - python >=3.5
+
+test:
+  source_files:
+    - tests
+  requires:
+    - pip
+    - coverage
+  imports:
+    - pytest_xprocess
+    - xprocess
+  commands:
+    - pip check
+    - coverage run -m pytest -v
+    - coverage report --fail-under={{ cov_fail_under }}
```

Comment:
I'm not sure running coverage tests in the feedstock are worth our cycles. This is something for devs upstream, we are not really changing the code (much) here.

Reply:
happy to have the broader discussion, but my motivation has been, especially on more complicated, multi-output packages, that the test suite is good at finding underlying required dependency issues, while coverage helps find drastic optional dependency changes that might trigger a new output.

Reply:
CI resources are OK now. My old mind is still plagued by our CircleCI/Travis-CI times. It is probably OK to run that in the feedstock.

Reply:
yeah, maybe there's a reasonable threshold under which, it's fine if do more validation. most noarch python would fall there... but being picky has definitely resulted in some upstream fixes.